plugins/foreman_discovery/15.0/index.md

```diff
@@ -990,10 +990,9 @@ option, run:
     fdi.script=H4sIAK4hZ1oCA0tNzshXUA/JyCxWAKJEhZLU4hJ1LgA92U9qFgAAAA==
 
 Keep in mind that kernel command line option is limited, Red Hat Enterprise
-Linux 7 (or CentOS 7) has limitation of 512 characters and there are already a
-lot of characters, so in practice the script option must be shorter than 300
-characters in total. When it exceeds the threshold, rest of the kernel command
-line option will be ignored, therefore it will likely fail.Typically the script
+Linux 7 (or CentOS 7) has limitation of 2048 characters and there are already a
+lot of characters, so in practice the script option must be shorter than this in total. When it exceeds the threshold, rest of the kernel command
```

Comment:
I realize it was already already there, but:
```suggestion
lot of characters, so in practice the script option must be shorter than this in total. When it exceeds the threshold, the rest of the kernel command
```

The rest of the file uses short lines with hard breaks. Is this something you want to keep?

Reply:
It's weird. I did not touch anything in that regard and the lines render in my editor without this long line. I'll add the break.

Reply:
Done thanks!